app/src/main/java/ch/epfl/sdp/peakar/points/ComputePOIPoints.java

```diff
@@ -98,11 +99,13 @@ private static void getPOIs(UserPoint userPoint){
         //Retrieve cache instance
         POICache poiCache = POICache.getInstance();
         //Check if file is present and if user is in BB
-        if(poiCache.isCacheFilePresent(ctx.getApplicationContext().getCacheDir()) && poiCache.isUserInBoundingBox(userPoint, ctx.getCacheDir()))
+        if(poiCache.isCacheFilePresent(ctx.getApplicationContext().getCacheDir()) && poiCache.isUserInBoundingBox(userPoint, ctx.getCacheDir())){
             getPOIsFromCache(userPoint);
-        else
+        }
+        else{
             Log.d("DEBUG", "Downmloaded");
```

Comment:
I see this is a debug comment, but maybe if you think it could be useful in future we can replace "DEBUG" with the class name, as it could be confusing while debugging (even other things) to see general "debug" prints.
```suggestion
            Log.d("ComputePOIPoints", "Downloaded");
```

Reply:
This log was removed as it is not needed anymore